models/zeroex/ethereum/zeroex_api_ethereum_fills.sql

```diff
@@ -0,0 +1,490 @@
+{{  config(
+        alias='fills',
+        materialized='incremental',
+        partition_by = ['block_date'],
+        unique_key = ['tx_hash', 'evt_index'],
+        on_schema_change='sync_all_columns',
+        file_format ='delta',
+        incremental_strategy='merge',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                "project",
+                                "zeroex",
+                                \'["sui414", "bakabhai993", "Hosuke"]\') }}'
+    )
+}}
+
+{% set zeroex_v3_start_date = '2019-12-01' %}
+{% set zeroex_v4_start_date = '2021-01-06' %}
```

Comment:
Hi @sui414, I have added the project_start_date for zeroex v3 and v4, these dates will be used in full-refresh for performance gain. There are some CTEs I am not sure which date should I use, so I use the default v3 date.

These dates are based on [the query](https://dune.com/queries/1559388).

Reply:
thank you! seems accurate to me. I added you as a collaborator for this PR. 
gonna finish solving the test error on unique id and ask dune team for Merge soonish.

Reply:
updated some of the subqueries' dates